taxonomies/labels.txt

```diff
@@ -17554,6 +17560,12 @@ opposite:en: en:vegan
 en:EcoVeg
 xx:EcoVeg
 
+<en:Vegan
+en:VegeCert - Certified Vegan
+
+<en:Vegan
+en:Certified iVegan
```

Comment:
```suggestion
en:Certified Vegan
```

Reply:
@stephanegigandet actually not a typo, it's "Certified iVegan" on the packaging. iMac, iPod, iVegan, I guess :-)

Reply:
ok, I found only 1 product: https://world.openfoodfacts.org/product/8011795103113/schiacciatelle-ai-cereali-forno-damiani
maybe Hunger Games will find more